modules/cnv-creating-local-block-pv.adoc

```diff
@@ -0,0 +1,62 @@
+// Module included in the following assemblies:
+//
+// * cnv/cnv_users_guide/cnv-importing-virtual-machine-images-datavolumes-block.adoc
+// * cnv/cnv_users_guide/cnv-cloning-vm-disk-into-new-datavolume-block.adoc
+
+[id="cnv-creating-local-block-pv_{context}"]
+= Creating a local block Persistent Volume
+
+Create a local block Persistent Volume (PV) on a node by initializing a file and mounting it as a loop device. This loop device can then be referenced in a PV configuration as a `Block` volume and used as a block device for a virtual machine image. 
```

Comment:
Create a local block Persistent Volume (PV) on a node by populating a file, and mounting it as a loop device. This loop device can then be referenced in a PV manifest as a `Block` volume, and used as a block device for a virtual machine image.